proto/streamlit/proto/DeckGlJsonChart.proto

```diff
@@ -28,7 +28,7 @@ message DeckGlJsonChart {
   // If True, will overwrite the chart width spec to fit to container.
   bool use_container_width = 4;
 
-  // the hash of the json so the the frontend doesn't always have to parse the pydeck json object
+  // ID
```

Comment:
nit: this comment probably does not provide too much added value over the already existing variable name 😁 I feel we should remove it

Reply:
Agreed! Will leave for now though, since this will change in upcoming PRs!